examples/examples/z_queryable.rs

```diff
@@ -40,9 +35,7 @@ async fn main() {
         .await
         .unwrap();
 
-    println!("Enter 'q' to quit, 'e' to reply an error to next query...");
-    let mut stdin = async_std::io::stdin();
-    let mut input = [0_u8];
+    println!("Press CTRL-C to quit...");
     loop {
         select!(
```

Comment:
Select is no longer necessary, just have a `while let Ok(query) = queryable.recv().await { .. }`.